spec/requests/static_request_spec.rb

```diff
@@ -1,34 +1,37 @@
 require "rails_helper"
 
-describe StaticController, type: :controller do
+describe "Static API Requests", type: :request do
   context "when getting the root url" do
     describe "GET #index" do
       it "returns http success" do
-        get :index
+        get root_path
         expect(response).to have_http_status(200)
       end
     end
   end
   context "when url is correct and template is available" do
     describe "GET #page" do
+      let(:name) { "index" }
```

Comment:
Im confused as to why this variable is defined in a `let` block. It doesnt appear to be re-used anywhere. Would it be better served to just define it as a plain old ruby variable in the `it` block below?

Reply:
This is a bit subjective. I think it makes the spec easier to read. I like the organization of defining all my dependent objects with let and keeping my it block nice and short.

Reply:
Agreed on the subjective nature of this comment. This isnt really a blocker to getting this merged 👍